wrapanapi/entities/physical_container.py

```diff
@@ -0,0 +1,30 @@
+# coding: utf-8
+"""
+wrapanapi.entities.physical_container
+
+Implements classes and methods related to physical entities such as racks,
+blocks, sleds, chassis or enclosures that contain other physical entities such
+as physical servers.
+"""
+from abc import ABCMeta
+
+from wrapanapi.entities.base import Entity
+
+
+class PhysicalContainer(Entity):
```

Comment:
Maybe it would be nice to handle python 2/3 compatibility here since this is a new file? Something like this should work (if we import `six`, of course):

```suggestion
class PhysicalContainer(six.with_metaclass(ABCMeta, Entity)):
```

This way we can remove the python2-only `__metaclass__` assignment.

Reply:
👍